docs/standard/garbage-collection/implementing-disposeasync.md

```diff
@@ -121,7 +121,7 @@ In the preceding example, each asynchronous clean up operation is implicitly sco
 
 ### Unacceptable pattern
 
-The highlighted lines in the following code show what it means to have "stacked usings". If an exception is thrown from the `AnotherAsyncDisposable` constructor, then `objOne` does not get properly disposed.
+The highlighted lines in the following code show what it means to have "stacked usings". If an exception is thrown from the `AnotherAsyncDisposable` constructor, both objects are not properly disposed of.
```

Comment:
I'd state this a bit differently. The key point is that if `objTwo`'s constructor throws an exception, the constructor must clean up any resources allocated before it throws:

```suggestion
The highlighted lines in the following code show what it means to have "stacked usings". If an exception is thrown from the `AnotherAsyncDisposable` constructor, neither object is properly disposed of. The variable `objTwo` is never assigned because the constructor did not complete successfully. As a result, the constructor for `AnotherAsyncDisposable` is responsible for disposing any resources allocated before it throws an exception. 
```

Reply:
Yes, I do like that - thank you!